icu4c/source/test/intltest/compactdecimalformattest.cpp

```diff
@@ -487,10 +487,10 @@ void CompactDecimalFormatTest::TestBug12975() {
     LocalPointer<CompactDecimalFormat> cdf(CompactDecimalFormat::createInstance(locale, UNUM_SHORT, status));
     if (assertSuccess("", status, true, __FILE__, __LINE__)) {
         UnicodeString resultCdf;
-        cdf->format(120000, resultCdf);
+        cdf->format(12000, resultCdf);
```

Comment:
So it looks like they added compact formatting but only for 100K and above.  Is this intentional?  It seems weird that `K` is used here but real Italian abbreviations are used for 10e6 and higher.

it.xml has:

```
<pattern type="1000" count="one">0</pattern>
<pattern type="1000" count="other">0</pattern>
<pattern type="10000" count="one">0</pattern>
<pattern type="10000" count="other">0</pattern>
<pattern type="100000" count="one">000K</pattern>
<pattern type="100000" count="other">000K</pattern>
<pattern type="1000000" count="one">0 Mln</pattern>
<pattern type="1000000" count="other">0 Mln</pattern>
<pattern type="10000000" count="one">00 Mln</pattern>
<pattern type="10000000" count="other">00 Mln</pattern>
<pattern type="100000000" count="one">000 Mln</pattern>
<pattern type="100000000" count="other">000 Mln</pattern>
<pattern type="1000000000" count="one">0 Mrd</pattern>
<pattern type="1000000000" count="other">0 Mrd</pattern>
<pattern type="10000000000" count="one">00 Mrd</pattern>
<pattern type="10000000000" count="other">00 Mrd</pattern>
<pattern type="100000000000" count="one">000 Mrd</pattern>
<pattern type="100000000000" count="other">000 Mrd</pattern>
<pattern type="1000000000000" count="one">0 Bln</pattern>
<pattern type="1000000000000" count="other">0 Bln</pattern>
<pattern type="10000000000000" count="one">00 Bln</pattern>
<pattern type="10000000000000" count="other">00 Bln</pattern>
<pattern type="100000000000000" count="one">000 Bln</pattern>
<pattern type="100000000000000" count="other">000 Bln</pattern>
```

Reply:
Again, I am not sure. Please file a CLDR bug and we can check with vetters about this.